h/api/search.py

```diff
@@ -7,12 +7,34 @@
 import logging
 
 import webob.multidict
+from annotator import document
```

Comment:
This should be `from h.api import models`. The fact that the `Document` model comes from `annotator` is an implementation detail.